src/components/Base/Link/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { translate } from 'react-i18next';
+
+import links from 'config/doc-link';
+
+@translate()
+export default class DocLink extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    isExternal: PropTypes.bool,
+    name: PropTypes.string,
+    to: PropTypes.string
+  };
+
+  static defaultProps = {
+    children: null,
+    className: '',
+    isExternal: true,
+    name: ''
+  };
+
+  render() {
+    const {
+      t, to, name, children, className, isExternal
+    } = this.props;
+    let text = t(`LINK_${name}`);
```

Comment:
No need to add `t()` here, since you compare text with `LINK_${name}`

Reply:
I want a default text from it props of `name` to `t(`LINK_${name})`